mpags-cipher.cpp

```diff
@@ -0,0 +1,50 @@
+#include <vector>
+#include <string.h>
+#include <iostream>
+#include <stdio.h>
+
+int main(int argc, char* argv[]) {
+
+/*
+A simple C++ tool to encrypt/decrypt text
+*/
+
+std::string version = "0.1.0";
+
+const std::vector<std::string> cmdLineArgs { argv, argv+argc };
+
+for (int i = 0; i < cmdLineArgs.size(); i++){
```

Comment:
When I compiled I got a warning about this line because you're comparing a signed type (`int`) with an unsigned type (`long unsigned int`, which is what is returned by `cmdLineArgs.size()`).
Indeed, the size of a container cannot be negative so it makes sense to use unsigned types here.
There is a helpful type `std::size_t` (which is a typedef for `long unsigned int`) that is very convenient to use in these situations.

NB also that you can start the loop from 1 since the 0th argument is the name of the executable, which we are not interested in.

```suggestion
for (std::size_t i{1}; i < cmdLineArgs.size(); i++){
```